test/eventHandlers/workerInitRequest.test.ts

```diff
@@ -84,4 +85,33 @@ describe('workerInitRequest', () => {
             },
         });
     });
+
+    it('correctly loads package.json file', async () => {
+        const appDir = 'appDir';
+        const expectedPackageJson = {
+            type: 'module',
+        };
+        mock({
+            [appDir]: {
+                'package.json': JSON.stringify(expectedPackageJson),
+            },
+        });
+        await channel.initAppDir(appDir);
+        expect(channel.functionAppDir).to.equal(appDir);
+        expect(channel.packageJson).to.deep.equal(expectedPackageJson);
+        mock.restore();
+    });
+
+    it('loads empty package.json', async () => {

```

Comment:
Add another test with a poorly-formed package.json (aka not json-parseable) just to make sure it doesn't block the worker from initializing

Reply:
Done